backend-comparison/benches/custom_gelu.rs

```diff
@@ -24,7 +24,16 @@ impl<B: Backend, const D: usize> Benchmark for CustomGeluBenchmark<B, D> {
     type Args = Tensor<B, D>;
 
     fn name(&self) -> String {
-        format!("Gelu {:?}", self.kind)
+        let kindstr = format!("{:?}", self.kind);
+        format!("gelu_{}", kindstr.to_lowercase())
```

Comment:
This should be "gelu", the kind is included in the options

Reply:
Ahah, ok I was expecting this one, I hesitated to do it. I did this to get a better filename but that's more a cosmetic tweak than a useful one.